samples/isilon_v250_k8s_121.yaml

```diff
@@ -0,0 +1,230 @@
+apiVersion: storage.dell.com/v1
```

Comment:
Please remove all samples related to v2.2.0

Reply:
Removed